sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.4.0b3 (Unreleased)
 
+- Preview of `VSCodeCredential` #10472
```

Comment:
This still calls out `VSCodeCredential`, the internal type no user should think about. The sentence below makes a better bullet. Also, please make the issue reference a link.